pkg/collector/corechecks/cluster/kubernetesapiserver/events_common_test.go

```diff
@@ -47,6 +49,59 @@ func TestGetDDAlertType(t *testing.T) {
 	}
 }
 
+func Test_getInvolvedObjectTags(t *testing.T) {
+	taggerInstance := local.NewFakeTagger()
+	taggerInstance.SetTags("namespace://default", "workloadmeta-kubernetes_node", []string{"team:container-int"}, nil, nil, nil)
+	tests := []struct {
+		name           string
+		involvedObject v1.ObjectReference
+		tags           []string
+	}{
+		{
+			name: "get pod basic tags",
+			involvedObject: v1.ObjectReference{
+				Kind:      "Pod",
+				Name:      "my-pod",
+				Namespace: "my-namespace",
+			},
+			tags: []string{
+				"kube_kind:Pod",
+				"kube_name:my-pod",
+				"kubernetes_kind:Pod",
+				"name:my-pod",
+				"kube_namespace:my-namespace",
+				"namespace:my-namespace",
+				"pod_name:my-pod",
+			},
+		},
+		{
+			name: "get pod namespace tags",
+			involvedObject: v1.ObjectReference{
+				Kind:      "Pod",
+				Name:      "my-pod",
+				Namespace: "default",
+			},
+			tags: []string{
+				"kube_kind:Pod",
+				"kube_name:my-pod",
+				"kubernetes_kind:Pod",
+				"name:my-pod",
+				"kube_namespace:default",
+				"namespace:default",
+				"team:container-int",
+				"pod_name:my-pod",
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := getInvolvedObjectTags(tt.involvedObject, taggerInstance); !reflect.DeepEqual(got, tt.tags) {
+				t.Errorf("getInvolvedObjectTags() = %v, want %v", got, tt.tags)
```

Comment:
can we use `assert.ElementsMatch` directly instead of providing a custom error? I don't think the custom error adds much extra info in this case.